src/LayerEditor/ui/data/tools/selector.py

```diff
@@ -1,19 +1,33 @@
-from PyQt5.QtCore import pyqtSignal
+from PyQt5.QtCore import pyqtSignal, QObject
 
 
-class Selector:
+class Selector(QObject):
     """ Used for selecting one item(value) from list.
         Also used to check if selected item is still in iterable object.
         If not than select other item that is part of iterable"""
+    value_changed = pyqtSignal()
+    """signal is emitted whenever value is changed"""
+
     def __init__(self, value):
-        self.value = value
+        super(Selector, self).__init__()
+        self._value = value
         self.old_value = None
 
+    @property
+    def value(self):
+        return self._value
+
+    @value.setter
```

Comment:
What is the use case for old_value? Seems that it is used for clean-up after the selector is set to a new value. 
That can be replaced by a setter, that returns the old value.

Reply:
Problém je že při změně hodnoty se vyšle signál. Pokusím se dát old_value do signálu. Tím by se to mělo vyřešit.